backend/de.metas.util/src/main/java/de/metas/util/collections/CollectionUtils.java

```diff
@@ -492,4 +492,11 @@ else if (excludes == null || excludes.isEmpty())
 			}
 		}
 	}
+
+	public static boolean hasDuplicatesForValue(@NonNull final Collection<String> collection, @NonNull final String value)
+	{
+		return collection.stream()
+				.filter(elem -> elem.equals(value))
```

Comment:
```suggestion
				.filter(elem -> value.equals(elem))
```

..value can't be null, but elem might be..